charts/incubator/mineos/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 kubeVersion: ">=1.16.0-0"
 name: mineos
-version: 0.0.1
-appVersion: "latest" #1.35.1
+version: "0.0.1"
+appVersion: "latest"
```

Comment:
stop fucking around with these please.
One is not a string and the other is autogenerated.

It's issues like these that are delaying your PR.
Please limit yourself to PR and CI feedback.